src/build.c

```diff
@@ -1278,27 +1261,25 @@ static void set_file_error_from_errno(GError **error, gint err, const gchar *pre
 
 
 /* write a little shellscript to call the executable (similar to anjuta_launcher but "internal")
- * fname is the full file name (including path) for the script to create */
-static gboolean build_create_shellscript(const gchar *fname, const gchar *working_dir, const gchar *cmd, gboolean autoclose, GError **error)
+ * it returns the full file name (including path) of the created script in the locale encoding */
+static gchar *build_create_shellscript(const gchar *working_dir, const gchar *cmd, gboolean autoclose, GError **error)
```

Comment:
However, we should write the script in locale encoding, so everythijg ending up there should be converted as appropriate.

*however*: on Windows cmd.exe uses a specific code page, so anything non-ASCII will probably fail.  We may need to set the codepage at the start of the script, but to what?